<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CfgTabPageUserHbciUi</class>
 <widget class="QWidget" name="CfgTabPageUserHbciUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit User</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="Q3GroupBox" name="groupBox9">
     <property name="title">
      <string>General HBCI Settings</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="1">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QComboBox" name="userStatusCombo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="finishUserButton">
          <property name="text">
           <string>Finish</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel6">
        <property name="text">
         <string>User Status</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel4">
        <property name="text">
         <string>Server URL</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="serverEdit">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3GroupBox" name="httpBox">
     <property name="title">
      <string>PIN/TAN Settings</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>HTTP protocol version</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel3">
        <property name="text">
         <string>HTTP User Agent</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="userAgentEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>iTAN Mode</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="httpVersionCombo"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="itanModeCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3GroupBox" name="groupBox2">
     <property name="title">
      <string>Expert Settings</string>
     </property>
     <layout class="QGridLayout">
      <item row="1" column="0">
       <widget class="QCheckBox" name="bankCounterCheck">
        <property name="text">
         <string>Bank uses signature counter</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="bankSignCheck">
        <property name="text">
         <string>Bank signs its messages</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="forceSsl3Check">
        <property name="toolTip">
         <string>Force the use of SSLv3</string>
        </property>
        <property name="whatsThis">
         <string>&lt;qt&gt;Force the use of &lt;b&gt;SSLv3&lt;/b&gt; in &lt;b&gt;Pin/Tan&lt;/b&gt; mode&lt;/qt&gt;</string>
        </property>
        <property name="text">
         <string>Force SSLv3</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="noBase64Check">
        <property name="toolTip">
         <string>Force the use of SSLv3</string>
        </property>
        <property name="whatsThis">
         <string>&lt;qt&gt;Force the use of &lt;b&gt;SSLv3&lt;/b&gt; in &lt;b&gt;Pin/Tan&lt;/b&gt; mode&lt;/qt&gt;</string>
        </property>
        <property name="text">
         <string>No BASE64 Encoding</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3GroupBox" name="expertActionsBox">
     <property name="title">
      <string>Expert Actions</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QPushButton" name="getServerKeysButton">
        <property name="text">
         <string>Get Server Keys</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="getSysIdButton">
        <property name="text">
         <string>Get System Id</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="getAccountsButton">
        <property name="text">
         <string>Get Accounts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="getItanModesButton">
        <property name="text">
         <string>Get iTAN Modes</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer27">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>21</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>userStatusCombo</tabstop>
  <tabstop>finishUserButton</tabstop>
  <tabstop>serverEdit</tabstop>
  <tabstop>httpVersionCombo</tabstop>
  <tabstop>userAgentEdit</tabstop>
  <tabstop>itanModeCombo</tabstop>
  <tabstop>bankSignCheck</tabstop>
  <tabstop>bankCounterCheck</tabstop>
  <tabstop>forceSsl3Check</tabstop>
  <tabstop>noBase64Check</tabstop>
  <tabstop>getServerKeysButton</tabstop>
  <tabstop>getSysIdButton</tabstop>
  <tabstop>getAccountsButton</tabstop>
  <tabstop>getItanModesButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
